src/index.js

```diff
@@ -0,0 +1,7 @@
+import './style.css';
+// import todoUI from '../modules/todoUi.js';
```

Comment:
- [OPTIONAL] Please kindly remove `commented code` from your `code base`, as it make your `code base` dirty and less readable.